api/v1/curvecluster_types.go

```diff
@@ -23,46 +23,41 @@ import (
 
 const CustomResourceGroup = "curve.opencurve.io"
 
-// ConditionType represent a resource's status
+// ConditionType represents a resource's status
 type ConditionType string
 
 const (
-	// ClusterPhasePending: The cluster is running to create.
-	// ClusterPhaseReady: The cluster has been created successfully.
-	// ClusterPhaseDeleting: The cluster is running to delete.
-	// ClusterPhaseError: The cluster created failed becasue some reason.
-	// ClusterPhaseUnknown: Unknow phase
-	// ClusterPending
+	// ClusterPhasePending : The cluster is running to create.
 	ClusterPhasePending ConditionType = "Pending"
-	// ClusterReady
+	// ClusterPhaseReady : The cluster has been created successfully.
```

Comment:
you would remove the colon will be more standardized

Reply:
I'll polish these comments later.

Reply:
PTAL